build-tools/packages/build-cli/src/commands/release.ts

```diff
@@ -57,6 +58,17 @@ export class ReleaseCommand<T extends typeof ReleaseCommand.flags> extends State
         const [context] = await Promise.all([this.getContext(), this.initMachineHooks()]);
         const flags = this.processedFlags;
 
+        // oclif doesn't support nullable boolean flags, so this works around that limitation by checking the args
+        // passed into the command. If neither are passed, then the default is determined by the branch config.
+        const branchPolicyCheckDefault = branchesRunDefaultPolicy.includes(
+            context.originalBranchName,
+        );
+        const userPolicyCheckChoice = this.argv.includes("--policyCheck")
+            ? true
+            : this.argv.includes("--no-policyCheck")
+            ? false
+            : undefined;
```

Comment:
style nit:
```suggestion
        const userPolicyCheckChoice = this.argv.includes("--policyCheck")
            ? true
            : this.argv.includes("--no-policyCheck")
                ? false
                : undefined;
```

Reply:
Or:
```suggestion
        const userPolicyCheckChoice = this.argv.includes("--policyCheck") ||
            (this.argv.includes("--no-policyCheck")
                ? false
                : undefined);
```

Ok spent enough time on trivia moving on!

Reply:
Welcome to the world of prettier-formatted code. Your opinion no longer matters. 😝

Reply:
![image](https://c.tenor.com/DTyy7H1JGZEAAAAC/nooooo-shout.gif)

Can we tweak that prettier rule? 😝

Reply:
I welcome you to [check their docs](https://prettier.io/docs/en/options.html), but I don't think so.